docs_nnx/guides/gemma.ipynb

```diff
@@ -45,13 +53,13 @@
    "cell_type": "markdown",
    "metadata": {},
    "source": [
-    "## Downloading the checkpoint\n",
+    "## Download the model\n",
     "\n",
-    "\"To use Gemma's checkpoints, you'll need a Kaggle account and API key. Here's how to get them:\n",
+    "To use Gemma model, you'll need a [Kaggle](https://www.kaggle.com/models/google/gemma/) account and API key:\n",
     "\n",
-    "1. Visit https://www.kaggle.com/ and create an account.\n",
-    "2. Go to your account settings, then the 'API' section.\n",
-    "3. Click 'Create new token' to download your key.\n",
+    "1. To create an account, visit [Kaggle](https://www.kaggle.com/) and click on 'Register'.\n",
+    "2. If/once you have an account, you need to sign in, go to your ['Settings'](https://www.kaggle.com/settings), and under 'API' click on 'Create New Token' to generate and download your Kaggle API key.\n",
+    "3. OPTIONAL: In [Google Colab](https://colab.research.google.com/), under 'Secrets' add your Kaggle username and API key, storing the username as `KAGGLE_USERNAME` and the key as `KAGGLE_KEY`. If you are using a [Kaggle Notebook](https://www.kaggle.com/code) for free TPU or other hardware acceleration, it has a key storage feature under 'Add-ons' > 'Secrets', along with instructions for accessing stored keys.\n",
```

Comment:
@cgarciae Added this step. Will remove "OPTIONAL" since Colab asks for access here after running the code below, so users won't have to manually entering the details if they are stored in Colab:

```
import kagglehub
kagglehub.login()
```